ArduCopter/GCS_Mavlink.cpp

```diff
@@ -695,6 +695,32 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_int_packet(const mavlink_command_i
 
     case MAV_CMD_DO_REPOSITION:
         return handle_command_int_do_reposition(packet);
+
+    // pause or resume an auto mission
+    case MAV_CMD_DO_PAUSE_CONTINUE:
```

Comment:
The code in here should be factored out into a `handle_command_pause_continue` method, and also be available from `handle_command_long_packet`.